tests/Widget/NoExistTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Tests\Widget;
+
+use InvalidArgumentException;
+use Yiisoft\Form\Tests\TestCase;
+use Yiisoft\Form\Tests\Stub\LoginForm;
+use Yiisoft\Form\Widget\PasswordInput;
+use Yiisoft\Form\Widget\TextInput;
+
+final class NoExistTest extends TestCase
+{
+    public function testNoExistWidget(): void
+    {
+        $data = new LoginForm();
+
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Undefined property: "Yiisoft\Form\Tests\Stub\LoginForm::undefined_form_attribute".');
+
+        (new TextInput)->config($data, 'login')->run();
```

Comment:
Why did you do this and next one line? 
It should be removed

Reply:
fix